modules/imap_folders/modules.php

```diff
@@ -129,6 +129,12 @@ public function process() {
  */
 class Hm_Handler_process_accept_special_folders extends Hm_Handler_Module {
     public function process() {
+        
+        if (!$this->module_is_supported('imap_folders')) {
+            Hm_Msgs('ERRIMAP folders module is not enabled');
```

Comment:
I think we can just return as users won't have a clue on this or add it or debug instead.